math/matrix/strassenmatrixmultiply_test.go

```diff
@@ -0,0 +1,91 @@
+package matrix
```

Comment:
Tests should be black box, i.e. under the package `matrix_test` cause you don't want to deal with internal stuff here.

Reply:
Updated All the **__test.go_** files under package `matrix_test`.